src/main.ts

```diff
@@ -3,6 +3,7 @@ import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  app.setGlobalPrefix('api');
```

Comment:
Adding global API prefix `/api`. It should be noted that this affects e2e tests because method test functions need to be called without the global prefix because it is implicitly applied; e.g: `get('/data')` not `get('/api/data')`.